platform/ui/src/components/studyBrowser/StudyBrowser.js

```diff
@@ -1,62 +1,102 @@
-    supportsDragAndDrop: PropTypes.bool.isRequired,
-    onThumbnailClick: PropTypes.func,
-    onThumbnailDoubleClick: PropTypes.func,
-  };
+  return (
+    <div className="study-browser">
+      <div className="scrollable-study-thumbnails">
+        {studies
+          .map((study, studyIndex) => {
+            const { studyInstanceUid } = study;
+            return study.thumbnails.map((thumb, thumbIndex) => {
+              // TODO: Thumb has more props than we care about?
+              const {
+                altImageText,
+                displaySetInstanceUid,
+                imageId,
+                instanceNumber,
+                numImageFrames,
+                seriesDescription,
+                seriesNumber,
+                stackPercentComplete,
+              } = thumb;
 
-  render() {
-    const studies = this.props.studies;
-
-    const thumbnails = studies.map((study, studyIndex) => {
-      return study.thumbnails.map((thumb, thumbIndex) => {
-        if (this.props.supportsDragAndDrop) {
-          return (
-            <ThumbnailEntryDragSource
-              key={thumb.displaySetInstanceUid}
-              {...study}
-              {...thumb}
-              id={`${studyIndex}_${thumbIndex}`}
-              onClick={this.props.onThumbnailClick}
-              onDoubleClick={this.props.onThumbnailDoubleClick}
-            />
-          );
-        } else {
-          return (
-            <div className="ThumbnailEntryContainer" data-cy="thumbnail-list">
-              <ThumbnailEntry
-                key={thumb.displaySetInstanceUid}
-                {...study}
-                {...thumb}
-                id={`${studyIndex}_${thumbIndex}`}
-                onClick={this.props.onThumbnailClick}
-                onDoubleClick={this.props.onThumbnailDoubleClick}
-              />
-            </div>
-          );
-        }
-      });
-    });
-
-    const components = thumbnails.flat();
-    return (
-      <div className="StudyBrowser">
-        <div className="scrollable-study-thumbnails">{components}</div>
+              return (
+                <div
+                  key={`container_${studyIndex}_${thumbIndex}`}
+                  className="thumbnail-container"
+                >
+                  <Thumbnail
+                    supportsDrag={supportsDrag}
+                    key={`${studyIndex}_${thumbIndex}`}
+                    id={`${studyIndex}_${thumbIndex}`} // Unused?
+                    // Study
+                    studyInstanceUid={studyInstanceUid} // used by drop
+                    // Thumb
+                    altImageText={altImageText}
+                    imageId={imageId}
+                    instanceNumber={instanceNumber}
+                    displaySetInstanceUid={displaySetInstanceUid} // used by drop
+                    numImageFrames={numImageFrames}
+                    seriesDescription={seriesDescription}
+                    seriesNumber={seriesNumber}
```

Comment:
You nixed `data-cy=thumbnail-list` from the thumbnail container. This is causing E2E test failures.